daml-lf/scenario-interpreter/BUILD.bazel

```diff
@@ -13,11 +13,15 @@ load(
     "//rules_daml:daml.bzl",
     "daml_compile",
 )
+load("@scala_version//:index.bzl", "scala_major_version")
 
 da_scala_library(
     name = "scenario-interpreter",
     srcs = glob(["src/main/**/*.scala"]),
     main_class = "com.daml.lf.speedy.Main",
+    scala_deps =
+        ["@maven//:org_typelevel_paiges_core"] +  #
+        (["@maven//:org_scalaz_scalaz_core"] if scala_major_version == "2.12" else []),
```

Comment:
Why a conditional dependency on Scalaz?

Reply:
scala 2.12 complains when the dependency is missing
scala 2.13 complains when the dependency is there.

Reply:
scala 2.12 complain about missing dependencies when missing.
scala 2.13 complain about unused dependencies when I add the dependencies

Reply:
can we use `versioned_scala_deps`?

Reply:
👆🏻